Sources/App/Models/Entities/Migrations/CreateAppUser.swift

```diff
@@ -8,7 +8,9 @@ struct CreateAppUser: Migration {
             .schema(AppUser.schema)
             .id()
             .field(.email, .string, .required)
+            .field(.name, .string, .required)
             .field(.hashedPassword, .string, .required)
+            .field(.shouldResetPassword, .bool, .required)
             .field(.createdAt, .datetime)
             .field(.updatedAt, .datetime)
             .unique(on: .id, .email)
```

Comment:
I think we should add an indexes on `shouldResetPassword` and `name`. It would also mean we could add the methods you created to the template so we all get them out of the box,

```swift
import FluentSQL
import Vapor

extension Database {
    private var assertSQL: SQLDatabase {
        guard let sql = self as? SQLDatabase else {
            fatalError("Cannot create index on non-SQL database")
        }
        return sql
    }

    func indexName(for fieldKey: FieldKey, onTable table: String) -> String {
        "\(table)_\(fieldKey)_idx"
    }

    func createIndex(for fieldKey: FieldKey, onTable table: String) -> EventLoopFuture<Void> {
        assertSQL
            .create(index: indexName(for: fieldKey, onTable: table))
            .on(table)
            .column(fieldKey.description)
            .run()
    }

    func dropIndex(for fieldKey: FieldKey, onTable table: String) -> EventLoopFuture<Void> {
        assertSQL
            .drop(index: indexName(for: fieldKey, onTable: table))
            .run()
    }
}
```

Reply:
@heidipuk I'll add it for name and email. Might make less sense for boolean field: https://stackoverflow.com/questions/42972726/postgresql-create-index-for-boolean-column (unless we do the partial index somehow?)

Reply:
good point 👍  I didn't know that about indexes on boolean, but makes great sense 😄